### PR TITLE
AV-84097: Fix copy to reflect updated snippet

### PR DESCRIPTION
**Pull request by ElliotFrancisHunter**
Fix for the ticket: https://couchbasecloud.atlassian.net/browse/AV-84097

Snippets have been updated but the copy for Android and Java was not.

### CODE REVIEW

File: modules/android/pages/working-with-vector-search.adoc [hunk 1]
```diff
@@ -36,10 +36,10 @@ include::java:example$snippets/common/main/java/com/couchbase/codesnippets/Vecto
 
 First, initialize the `config` variable with the `VectorIndexConfiguration()` method with the following parameters:
 
-* The width or `dimensions` of the vector index is set to `300`.
+* The width or `dimensions` of the vector index is set to `3`.
 
-* The amount of `centroids` is set to `20`.
-This means that there will be 20 buckets with a single centroid each that gathers together similar vectors.
+* The amount of `centroids` is set to `100`.
+This means that there will be 100 buckets with a single centroid each that gathers together similar vectors.
```
Comment (RichardSmedley) [inline modules/android/pages/working-with-vector-search.adoc:42]: Is there somewhere else in the mobile docs that defines `bucket` in opposition to regular Couchbase Server buckets?

Comment (ElliotFrancisHunter) [reply modules/android/pages/working-with-vector-search.adoc:42]: There is no Couchbase Lite definition of bucket on the mobile side no, was a term decided by PM

Comment (osfameron) [reply modules/android/pages/working-with-vector-search.adoc:42]: "bucket" is a fairly common name in algorithms... but that's a good point from Richard 🤔

**Review by osfameron — APPROVED** [review]
As a pure change, I think this is good to get the correction in.
Definitely worth making note of the overloaded term "buckets", and perhaps discussion on docs-style-and-tone and in future docs terminology discussions.

File: modules/android/pages/working-with-vector-search.adoc (hunk 1 above)
Comment (ElliotFrancisHunter) [reply modules/android/pages/working-with-vector-search.adoc:42]: I shall raise it with Iveta in the meantime, seems like a useful thing to discuss with Iveta.

**Pull request merged**